sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -95,4 +115,46 @@ public static synchronized void init(
       throw new RuntimeException("Failed to initialize Sentry's SDK", e);
     }
   }
+
+  /**
+   * Deduplicate potentially duplicated Fragment and Timber integrations, which can be added
+   * automatically by our SDK as well as by the user. The user's ones win over ours.
+   *
+   * @param options SentryOptions to retrieve integrations from
+   */
+  private static void deduplicateIntegrations(
```

Comment:
This will be a spot where we'll need to "remember" to check for new integrations we add in the future.
What about we dedupe when iterating the list before calling `register`?

Reply:
mm, we could, but what would be the benefit? even if we add a new integration, we do not touch the code around `register`, right? + we couldn't reuse the LoadClass result there which is costly on Android due to reflection.